lesson2/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
+      <ul id="js-list"></ul>
```

Comment:
Instead of naming the id `js-list` for `ul`, I thought it would be more natural to change it to the plural form like JS, or name the element "js-ul".

Reply:
Thank you.
I thought that would be more explicit.
I have made the correction.